compose.yaml

```diff
@@ -25,6 +25,8 @@ x-beeai-agent: &beeai-agent
     - .secrets/rhel-config.json:/home/beeai/rhel-config.json:ro,z,U
     - .secrets/jotnar-vertex-dev.json:/home/beeai/jotnar-vertex-dev.json:ro,z,U
   restart: unless-stopped
+  tty: true
+  stdin_open: true
```

Comment:
Debugging always worked for me without this.

Reply:
I dropped the commit